pycbc/workflow/datafind.py

```diff
@@ -952,10 +952,12 @@ def run_datafind_instance(cp, outputDir, connection, observatory, frameType,
     # directory to check if this was expected.
     log_datafind_command(observatory, frameType, startTime, endTime,
                          os.path.join(outputDir,'logs'), **dfKwargs)
-    logging.info("Asking datafind server for frames.")
-    dfCache = connection.find_frame_urls(observatory, frameType,
-                                        startTime, endTime, **dfKwargs)
-    logging.info("Frames returned")
+    logging.debug("Asking datafind server for frames.")
```

Comment:
I suggest keeping this to `logging.info` due to a recent change in default logging level which would make debug logging invisible.